static/js/common/upload-addon.js

```diff
@@ -327,12 +327,28 @@
 
                     $("<strong>").text(message).appendTo(upload_results);
 
+                    // Specific messages for unlisted addons.
+                    var isListed = $('#id_is_listed').is(':checked'),
+                        isSideload = $('#id_is_sideload').is(':checked');
+                    if (!isListed) {
+                      if (isSideload) {
+                        $("<p>").text(gettext("Your submission, as a sideloaded add-on, will go through a manual review.")).appendTo(upload_results);
```

Comment:
@jvillalobos I've put some screenshots in the [PR's description](https://github.com/mozilla/olympia/pull/503) to show where those messages go (they are circled in red on the screenshots).

Should I change the message to "Your submission will go through a manual review"?